llpc/patch/llpcPatchBufferOp.cpp

```diff
@@ -1756,7 +1756,10 @@ Value* PatchBufferOp::ReplaceLoadStore(
             }
             else
             {
-                pPart = m_pBuilder->CreateIntrinsic(Intrinsic::amdgcn_raw_buffer_load,
+                unsigned IntrID = Intrinsic::amdgcn_raw_buffer_load;
```

Comment:
unsigned IntrID -> uint32_t intrinsicID = ...

Reply:
Done and done.

Reply:
Done.